DigitalLearningSolutions.Data/DataServices/NotificationDataService.cs

```diff
@@ -105,5 +107,24 @@ public IEnumerable<int> GetRoleBasedNotifications(int isCentreManager, int isCon
                 ).AsEnumerable();
         }
 
+        public void SubscribeDefaultNotifications(int? candidateId, int? adminUserId)
+        {
+            if (candidateId != null)
+            {
+                connection.Execute(
+                @"INSERT INTO NotificationUsers (
+                    NotificationID,CandidateID)
+                    (SELECT NotificationID,@candidateId FROM NotificationRoles WHERE RoleID = 5)",new {candidateId});
+            }
+
+            if (adminUserId != null)
+            {
+                connection.Execute(
+                @"INSERT INTO NotificationUsers (

```

Comment:
The logic removed at line 237 of RegistrationDataService.cs is better than this, we should revert this code to continue to use that logic. I had assumed that the subscription to notifications had been missed entirely but I see now that it was only present for admin registrations.

Reply:
This is addressed.